app/src/main/java/org/sopt/androidseminar1/home/HomeActivityViewPagerAdapter.kt

```diff
@@ -0,0 +1,13 @@
+package org.sopt.androidseminar1.home
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class HomeActivityViewPagerAdapter(fragmentActivity: FragmentActivity) : FragmentStateAdapter(fragmentActivity) {
+    val fragments = mutableListOf<Fragment>()
+
+    override fun getItemCount(): Int = fragments.size
+
+    override fun createFragment(position: Int): Fragment = fragments[position]
```

Comment:
```suggestion
    override fun getItemCount() = fragments.size

    override fun createFragment(position: Int) = fragments[position]
```
반환 타입은 빼주셔도 되는데, 있으면 또 가독성이 올라가서 취향에 따라서 해주시면 될 것 같습니다!